website/static/js/project.js

```diff
@@ -262,6 +262,17 @@ $(document).ready(function() {
         });
     });
 
+    $('#citation-more').on('click', function() {
```

Comment:
Actually, is this used? I could not find any usage of the `citation-more` ID in the templates.